contracts/Archetype.sol

```diff
@@ -143,17 +147,6 @@ contract Archetype is
       signature
     );
 
-    {
-      uint32 unitSize = invite.unitSize;
-      for (uint256 i = 0; i < toList.length; i++) {
-        if (unitSize > 1) {
-          _mint(toList[i], quantityList[i] * unitSize);
-        } else {
-          _mint(toList[i], quantityList[i]);
-        }
-      }
-    }
-
```

Comment:
Nice simplification here, when I was initially writing this I was thinking logically through the steps (count quantity -> validate -> mint). Didn't think of minting in the initial loop.